### ISSUE TITLE
Error on ec2_on_demand

### ISSUE THREAD

**Issue by lecoueyl** [srbartlett/awscosts#9]
When performing this command:

```
region = AWSCosts.region 'us-east-1'
region.ec2.on_demand(:linux).price
```

This error occurs:
```
/usr/local/lib/ruby/gems/2.1.0/gems/awscosts-0.0.7/lib/awscosts/ec2_on_demand.rb:56:in `fetch': undefined method `[]' for nil:NilClass (NoMethodError)
        from /usr/local/lib/ruby/gems/2.1.0/gems/awscosts-0.0.7/lib/awscosts/ec2.rb:19:in `on_demand'
        from ./test.rb:41:in `<main>'
```

There is no issue when performing requests on reserved or ELB class!

**Comment by srbartlett** [comment]
Thanks for the headups.  I need to rerecord my VCR tests more frequently.

Fixed and pushed a new version: 0.0.8

**Comment by lecoueyl** [comment]
Nice! Good job!
